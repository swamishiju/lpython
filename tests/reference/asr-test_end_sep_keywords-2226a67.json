{
    "basename": "asr-test_end_sep_keywords-2226a67",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/test_end_sep_keywords.py",
    "infile_hash": "5ea30711228d4ebb64266988c1a706a3d64f196457b939ed3bf15ecf",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_end_sep_keywords-2226a67.stdout",
    "stdout_hash": "10979f1767ba1a273eb53790f65f046101b37fc5fe9ae271e44cf5ad",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}