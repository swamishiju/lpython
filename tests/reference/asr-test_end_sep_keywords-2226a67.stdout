(TranslationUnit
    (SymbolTable
        1
        {
            __main__:
                (Module
                    (SymbolTable
                        2
                        {
                            __main__global_stmts:
                                (Function
                                    (SymbolTable
                                        4
                                        {
                                            
                                        })
                                    __main__global_stmts
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    [testEndSepKeywords]
                                    []
                                    [(SubroutineCall
                                        2 testEndSepKeywords
                                        ()
                                        []
                                        ()
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            testEndSepKeywords:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            
                                        })
                                    testEndSepKeywords
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    [(Print
                                        (StringFormat
                                            ()
                                            [(StringConstant
                                                "abc"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "lmn"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "pqr"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )]
                                            FormatPythonFormat
                                            (Allocatable
                                                (String 1 () DeferredLength DescriptorString)
                                            )
                                            ()
                                        )
                                    )
                                    (Print
                                        (StringFormat
                                            ()
                                            [(StringConstant
                                                "abc"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "lmn"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "pqr"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )]
                                            FormatPythonFormat
                                            (Allocatable
                                                (String 1 () DeferredLength DescriptorString)
                                            )
                                            ()
                                        )
                                    )
                                    (Print
                                        (StringFormat
                                            ()
                                            [(StringConstant
                                                "abc"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "lmn"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "pqr"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )]
                                            FormatPythonFormat
                                            (Allocatable
                                                (String 1 () DeferredLength DescriptorString)
                                            )
                                            ()
                                        )
                                    )
                                    (Print
                                        (StringFormat
                                            ()
                                            [(StringConstant
                                                "abc"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "lmn"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )
                                            (StringConstant
                                                "pqr"
                                                (String 1 (IntegerConstant 3 (Integer 4) Decimal) ExpressionLength DescriptorString)
                                            )]
                                            FormatPythonFormat
                                            (Allocatable
                                                (String 1 () DeferredLength DescriptorString)
                                            )
                                            ()
                                        )
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                )
                        })
                    __main__
                    ()
                    []
                    .false.
                    .false.
                    .false.
                ),
            main_program:
                (Program
                    (SymbolTable
                        5
                        {
                            __main__global_stmts:
                                (ExternalSymbol
                                    5
                                    __main__global_stmts
                                    2 __main__global_stmts
                                    __main__
                                    []
                                    __main__global_stmts
                                    Public
                                )
                        })
                    main_program
                    [__main__]
                    [(SubroutineCall
                        5 __main__global_stmts
                        2 __main__global_stmts
                        []
                        ()
                    )]
                )
        })
    []
)
